Dockerfile

```diff
@@ -52,6 +52,9 @@ RUN cd / && pip install --no-cache /app
 ENV PYTHONPATH /app
 RUN chown -R app:app /app
 
+# Create repos directory for transplanting in landing-worker
+RUN mkdir /repos
```

Comment:
Do we need `VOLUME /repos` too?

Reply:
We just need the directory so that it can be mounted here. There is no data to be persisted in the container as is.